bctw-api/src/apis/map_api.ts

```diff
@@ -228,7 +228,8 @@ const getCritterTracks = async (
       jsonb_build_object (
         'type', 'Feature',
         'properties', json_build_object(
-          'critter_id', critter_id
+          'critter_id', critter_id,
+          'deployment_id', deployment_id
```

Comment:
Just to confirm, this is still totally backwards compatible with BCTW-UI?

Reply:
I have checked the db changes against live dev and it was totally fine, but just be extra sure I will check this api change against my local ui real quick before I hit merge.

Reply:
Yup, all good.